include/quicr/moq_message_types.h

```diff
@@ -0,0 +1,281 @@
+#pragma once
```

Comment:
The plan is  to have all these in the quicr namespace since eventually we will get rid of quicr messge types and use these .. Having it defined in its own namespace forces us to import yet another namespace.

Reply:
OTOH we do have some duplicate  naming on some of the messages.

Reply:
may be it might be worth doing .. let me think a bit more on this